components/atoms/Select/single-select.tsx

```diff
@@ -51,7 +54,11 @@ const SingleSelect = ({
       }}
     >
       <SelectTrigger
-        className="bg-white min-w-[200px] w-full border focus-within:border-orange-500 focus-within:ring-orange-100 focus-within:ring ring-light-slate-6"
+        data-inset-label={insetLabel}
+        className={clsx(
+          `bg-white border`,
+          insetLabel && `before:content-[attr(data-inset-label)] before:mr-1 before:font-normal before:text-slate-500`
```

Comment:
Only add the inset label to the combobox's [::before](https://developer.mozilla.org/en-US/docs/Web/CSS/::before) if the `insetLAbel` prop is set.